src/components/Atoms/Combobox.tsx

```diff
@@ -30,6 +30,8 @@ export const Combobox = (props: ComboboxProps) => {
   const [usingPanel, setUsingPanel] = createSignal(false);
   const [inputValue, setInputValue] = createSignal("");
 
+  let inputBox: HTMLInputElement | undefined = undefined;
```

Comment:
```suggestion
  const inputBox: HTMLInputElement | undefined = undefined;
```

Noticed the linter picked this one up

Reply:
This doesn't work, I tried this and it doesn't work, It seems that solid redefines it later when you set it as ref in a component

Reply:
Then let's just add the linter ignore comment. That's an interesting error. I wonder if there's a useref hook equivalent in solid.

Reply:
This apparently is the equivalent of useRef, personally not a fan of it because it feels like magic